MapboxCoreNavigation/NavigationServiceDelegate.swift

```diff
@@ -139,6 +139,17 @@ public protocol NavigationServiceDelegate: class, UnimplementedLogging {
      */
     func navigationService(_ service: NavigationService, shouldPreventReroutesWhenArrivingAt waypoint: Waypoint) -> Bool
     
+    /**
+     Called when the location manager's accuracy authorization changed.
+     
+     You can implement this method to allow the navigation service to check if the user changed accuracy authorization, especially if reducedAccuracy is enabled. This method is only relevant for iOS 14 and above.
+     
+     - parameter service: The navigation service that will alert that user that reducedAccuracy is enabled.
+     - parameter manager: The location manager.
+     - returns: A bool indicating whether to alert the user to enable preciseAccuracy.
+     */
+    func locationManagerDidChangeAuthorization(_ manager: CLLocationManager)
```

Comment:
Part of the delegate pattern is that the first part of the method name always names the delegator, so that a single class can serve as the delegate for multiple delegators. Following that pattern, this method would be named `navigationServiceDidChangeAuthorization(_:)` and take the NavigationService as the first argument, not the CLLocationManager. It would then fall to the delegate’s implementation of this method to get the location manager from the navigation service.